### ISSUE TITLE
[EN][Dragon Soul][Hagara]Focused Assault

### ISSUE THREAD

**Issue by stanq87** [Atlantiss/BugTracker#9836]
**Description**: 
Focused Assault's range is above melle range, and it ignores dodge parry block  mecanics 

**Current behaviour**: 
even if u run away from the bosses melle range the skill keeps damaging u , and  u cant dodge parry or blockt the skil

**Expected behaviour**: 
when u leave the melle range the dmg shoulde stop, and it should be possible to dodge parry block the atack
http://cata-shoot.tauri.hu/?spell=107851
http://www.wowhead.com/spell=107851

**Steps to reproduce the problem**:

1. take a ctc capped tank
2. make him tank the boss
3. 

**Revision**: rev. 5630

**Comment by mati865** [comment]
> Cannot be dodged, parried or blocked.

http://www.wowhead.com/spell=107851/focused-assault#comments:id=1620052

**Comment by stanq87** [comment]
and yet https://www.youtube.com/watch?v=qtPG2JpV0FI

**Comment by Psychobird7** [comment]
TLDR: Focused assault is not meant to be blockable, dodgeable or parriable. 

The following are logs of prot paladins tanking Hagara 10H.      
http://www.worldoflogs.com/reports/pauko9t62zllmsjm/details/7/?s=1984&e=2098 
http://www.worldoflogs.com/reports/zvj3bqptq4jl6dw1/details/1/?s=1922&e=2162 
Go into the "Damage by spell" tab. 
Scroll down to "Focused assault". You will see X hits, 0 blocks, 0 parries, 0 dodges.  

You could probably do the exact same with logs of prot warriors, which I will do to spare you the effort. 
http://www.worldoflogs.com/reports/rt-tu6nfcdzmrxejx3p/details/177/?s=8635&e=8757 
http://www.worldoflogs.com/reports/h0bb1a5vlu7b4iuq/details/9/?s=2580&e=2799 
Repeat the process as with the paladin logs, and see there is also not a single block, dodge or parry. 

Here are blood DKs. 
http://www.worldoflogs.com/reports/rt-tfp39c5ynlx7ygxx/details/1/?s=1814&e=1921 
http://www.worldoflogs.com/reports/vge75iya8zywfxhh/details/6/?s=3249&e=3365 

And finally, feral druids.  
http://www.worldoflogs.com/reports/rt-9fn5d87nygyayxd7/details/4/?s=3046&e=3233  
http://www.worldoflogs.com/reports/rt-43sez4y13vqt7a15/details/9/?s=1975&e=2091 

You can repeat these steps for any other difficulty, and find the same results.

**Comment by Psychobird7** [comment]
Moving out of range cancelling Focused Assault might be an LFR-only mechanic.     

However, Focused Assault should *not* be parriable, dodgeable or blockable. 
See the following logs. Go into the "Damage by spell" tab and look at "Focused assault".  
http://www.worldoflogs.com/reports/pauko9t62zllmsjm/details/7/?s=1984&e=2098 
http://www.worldoflogs.com/reports/kq7wdq9sacp649yi/details/3/?s=1728&e=1962

**Comment by kadi2** [comment]
25 HC seems to not work https://youtu.be/aguo5K5EK-o?t=32s

**Comment by Aival94** [comment]
In Normal the avoidance is working just fine(maybe there is sth about range but seems to work just fine. If there is any proof of this bug please make a comment either here or on https://github.com/Atlantiss/BugTracker/issues/9867 to implement it.
